src/utils/config.py

```diff
@@ -54,25 +66,39 @@
     "user": os.getenv("POCKET_PUBLISHER_DATABASE_USER"),
     "db_name": os.getenv("POCKET_PUBLISHER_DATABASE_DBNAME"),
     "port": int(_MYSQL_PUBLISHER_PORT) if _MYSQL_PUBLISHER_PORT else 3306,
-    "password": os.getenv('POCKET_PUBLISHER_DATABASE_PASSWORD'),
+    "password": os.getenv("POCKET_PUBLISHER_DATABASE_PASSWORD"),
 }
 
-SNOWFLAKE_SNOWPLOW_DB = os.getenv('SNOWFLAKE_SNOWPLOW_DB')
-SNOWFLAKE_SNOWPLOW_SCHEMA = os.getenv('SNOWFLAKE_SNOWPLOW_SCHEMA')
-SNOWFLAKE_RAWDATA_DB = os.getenv('SNOWFLAKE_RAWDATA_DB')
-SNOWFLAKE_RAWDATA_FIREHOSE_SCHEMA = os.getenv('SNOWFLAKE_RAWDATA_FIREHOSE_SCHEMA')
-SNOWFLAKE_SNAPSHOT_DB = os.getenv('SNOWFLAKE_SNAPSHOT_DB')
-SNOWFLAKE_SNAPSHOT_FIREHOSE_SCHEMA = os.getenv('SNOWFLAKE_SNAPSHOT_FIREHOSE_SCHEMA')
+SNOWFLAKE_SNOWPLOW_DB = os.getenv("SNOWFLAKE_SNOWPLOW_DB")
+SNOWFLAKE_SNOWPLOW_SCHEMA = os.getenv("SNOWFLAKE_SNOWPLOW_SCHEMA")
+SNOWFLAKE_RAWDATA_DB = os.getenv("SNOWFLAKE_RAWDATA_DB")
+SNOWFLAKE_RAWDATA_FIREHOSE_SCHEMA = os.getenv("SNOWFLAKE_RAWDATA_FIREHOSE_SCHEMA")
+SNOWFLAKE_SNAPSHOT_DB = os.getenv("SNOWFLAKE_SNAPSHOT_DB")
+SNOWFLAKE_SNAPSHOT_FIREHOSE_SCHEMA = os.getenv("SNOWFLAKE_SNAPSHOT_FIREHOSE_SCHEMA")
 
-ATHENA_S3_OUTPUT = os.getenv('PREFECT_S3_ATHENA_QUERY_OUTPUT_BUCKET')
+ATHENA_S3_OUTPUT = os.getenv("PREFECT_S3_ATHENA_QUERY_OUTPUT_BUCKET")
 
 
 SQS_MESSAGE_VERSION = 3
-SQS_REC_QUEUE = 'RecommendationAPI-Prod-Sqs-Translation-Queue' \
-    if ENVIRONMENT == ENV_PROD \
-    else 'RecommendationAPI-Dev-Sqs-Translation-Queue'
-SQS_PROSPECT_QUEUE = 'ProspectAPI-Prod-Sqs-Translation-Queue' \
-    if ENVIRONMENT == ENV_PROD \
-    else 'ProspectAPI-Dev-Sqs-Translation-Queue'
+SQS_REC_QUEUE = (
+    "RecommendationAPI-Prod-Sqs-Translation-Queue"
+    if ENVIRONMENT == ENV_PROD
+    else "RecommendationAPI-Dev-Sqs-Translation-Queue"
+)
+SQS_PROSPECT_QUEUE = (
+    "ProspectAPI-Prod-Sqs-Translation-Queue"
+    if ENVIRONMENT == ENV_PROD
+    else "ProspectAPI-Dev-Sqs-Translation-Queue"
+)
 
 
+ARTICLES_DB_SCHEMA_DICT = {
+    "local": {
+        "articles_database_name": os.getenv("SNOWFLAKE_DB"),
+        "articles_schema_name": os.getenv("SNOWFLAKE_DEV_SCHEMA"),
+    },
+    "production": {
+        "articles_database_name": "raw",
+        "articles_schema_name": "item",
+    },
+}
```

Comment:
Not specific to this PR: Our config is becoming increasingly complex due to different patterns being used, and a lack of structure. Python has many libraries that help manage configuration.

Reply:
I agree with this and think it would be best to apply better configuration options to our Prefect v2 work